ttter/accounts/tests.py

```diff
@@ -0,0 +1,81 @@
+from django.test import TestCase
+from django.urls import resolve, reverse
+
+from .models import MyUser
+# from models import MyUser
+from .views import UserDataInput
+
+
+class SignUpTests(TestCase):
+    def setUp(self):
+        self.url = reverse('accounts:user_data_input')
+
+    def test_signup_get(self):
+        response = self.client.get(self.url)
+        # ユーザーがURL/signup/のページをGETすれば、ステータスコードは200、すなわち’成功’となる。
+        self.assertEquals(response.status_code, 200)
+
+    def test_successful_signup_post(self):
+        data = {
+            'username': 'new_user',
+            'password1': 'testpass1',
+            'password2': 'testpass1',
+            'email': 'hogehoge.com',
+            'nickname': 'nu',
+            'date_of_birth': '2000/1/1'
+        }
+        post_response = self.client.post(self.url, data=data)
+        self.assertEquals(post_response.status_code, 200)
+        self.assertTemplateUsed(post_response, 'accounts/create.html')
+
+
+class UserDataConfirmTests(TestCase):
+    def setUp(self):
+        self.url = reverse('accounts:user_data_confirm')
+
+    def test_create_confirm(self):
+        data = {
+            'username': 'new_user',
+            'password1': 'testpass1',
+            'password2': 'testpass1',
+            'email': 'hogehoge.com',
+            'nickname': 'nu',
+            'date_of_birth': '2000/1/1'
+        }
+        response = self.client.post(self.url, data=data)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'accounts/create.html')
+
```

Comment:
```suggestion
self.assertContains(response, 'new_user')
```
例えば、このようにassertContainsを使用することでuser_data_confirmのviewにpostした際のcreate.htmlには'new_user'のテキストが含まれていることが確認できます。